answer.md

```diff
@@ -0,0 +1,24 @@
+
+Hi John Doe,
+
+Here are a list of potential security threats to consider for your proposed bank web application
```

Comment:
Nice use of lists - they're often a good way to make emails "punchy" so people actually read them.